static/js/protein/protein_protein_interaction_graph.tsx

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Protein-protein interaction graph
+ */
+
+import _ from "lodash";
+import React from "react";
+
+import { drawProteinInteractionGraph } from "./chart";
+import { getProteinInteractionGraphData } from "./data_processing_utils";
+import { InteractingProteinType } from "./page";
+import { MultiLevelInteractionGraphData } from "./types";
+
+type InteractionGraphProps = {
```

Comment:
usually we define the props and state as `interface InteractionGraphProps {...}`

Reply:
got it! done!